### PR TITLE
AUTH_LOGIN's resolve's result goes as payload on USER_LOGIN_SUCCESS

### PR DESCRIPTION
**Pull request by SidFerreira**
Based on #582, this small change will send the result of `AUTH_LOGIN`'s resolve as a payload on `USER_LOGIN_SUCCESS`. This way we can integrate, in a simple way, integrate the login with redux.
In my case I just wanted to save user's profile on the store.
Based on the discussion seem that there are some more complex ideas around but this change is enough for what I need and this is my suggestion.

( Replaces PR #615 )

### CODE REVIEW

File: src/sideEffect/saga/auth.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default (authClient) => {
             try {
                 yield put({ type: USER_LOGIN_LOADING });
                 yield call(authClient, AUTH_LOGIN, payload);
-                yield put({ type: USER_LOGIN_SUCCESS });
+                yield put({ type: USER_LOGIN_SUCCESS, payload });
```
Comment (fzaninotto) [inline src/sideEffect/saga/auth.js]: I think you'd better pass the result of the call from line 24 as the payload. That way, you grab whatever the `authClient` decides to expose (login and password, but also why not: tokens, roles, preferences, etc).

Comment (SidFerreira) [inline src/sideEffect/saga/auth.js]: I'll do that tonight when I check the date issues.

**Comment by SidFerreira** [comment]
@fzaninotto fixed

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**